ansys/mapdl/core/launcher.py

```diff
@@ -1041,6 +1052,60 @@ def launch_mapdl(
     additional_switches = _validate_add_sw(
         additional_switches, exec_file, kwargs.pop("force_intel", False)
     )
+
+    if license_type is not None and isinstance(license_type, str):
```

Comment:
This can be simplified because `None` is not a `str`.

```suggestion
    if isinstance(license_type, str):
```

Reply:
LGTM